go.mod

```diff
@@ -3,9 +3,46 @@ module github.com/submariner-io/coastguard
 go 1.12
 
 require (
+	cloud.google.com/go v0.43.0 // indirect
+	github.com/Shopify/sarama v1.23.0 // indirect
+	github.com/apache/thrift v0.12.0 // indirect
+	github.com/aws/aws-sdk-go v1.21.1 // indirect
+	github.com/coredns/coredns v1.4.0
+	github.com/coreos/bbolt v1.3.3 // indirect
+	github.com/coreos/etcd v3.3.13+incompatible // indirect
+	github.com/coreos/go-semver v0.3.0 // indirect
+	github.com/coreos/go-systemd v0.0.0-20190719114852-fd7a80b32e1f // indirect
+	github.com/coreos/pkg v0.0.0-20180928190104-399ea9e2e55f // indirect
+	github.com/dgrijalva/jwt-go v3.2.0+incompatible // indirect
+	github.com/dnstap/golang-dnstap v0.1.0 // indirect
+	github.com/farsightsec/golang-framestream v0.0.0-20190425193708-fa4b164d59b8 // indirect
+	github.com/flynn/go-shlex v0.0.0-20150515145356-3f9db97f8568 // indirect
+	github.com/gophercloud/gophercloud v0.2.0 // indirect
+	github.com/gorilla/websocket v1.4.0 // indirect
+	github.com/grpc-ecosystem/go-grpc-middleware v1.0.0 // indirect
+	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0 // indirect
+	github.com/grpc-ecosystem/grpc-gateway v1.9.4 // indirect
+	github.com/grpc-ecosystem/grpc-opentracing v0.0.0-20180507213350-8e809c8a8645 // indirect
+	github.com/jonboulle/clockwork v0.1.0 // indirect
+	github.com/mholt/caddy v0.11.5
+	github.com/miekg/dns v1.1.6
 	github.com/onsi/ginkgo v1.8.0
 	github.com/onsi/gomega v1.5.0
+	github.com/opentracing-contrib/go-observer v0.0.0-20170622124052-a52f23424492 // indirect
+	github.com/opentracing/opentracing-go v1.1.0 // indirect
+	github.com/openzipkin-contrib/zipkin-go-opentracing v0.3.5 // indirect
+	github.com/openzipkin/zipkin-go-opentracing v0.3.5 // indirect
+	github.com/prometheus/client_golang v1.0.0 // indirect
+	github.com/soheilhy/cmux v0.1.4 // indirect
 	github.com/submariner-io/submariner v0.0.0-20190708095718-350482d85dd4
+	github.com/tmc/grpc-websocket-proxy v0.0.0-20190109142713-0ad062ec5ee5 // indirect
+	github.com/xiang90/probing v0.0.0-20190116061207-43a291ad63a2 // indirect
+	go.etcd.io/bbolt v1.3.3 // indirect
+	go.uber.org/atomic v1.4.0 // indirect
+	go.uber.org/multierr v1.1.0 // indirect
+	go.uber.org/zap v1.10.0 // indirect
+	google.golang.org/grpc v1.22.0 // indirect
+	gopkg.in/jcmturner/goidentity.v3 v3.0.0 // indirect
 )
 
 replace github.com/bronze1man/goStrongswanVici => github.com/mangelajo/goStrongswanVici v0.0.0-20190223031456-9a5ae4453bd
```

Comment:
Is this needed, ie lighthouse doesn't use strongSwan?

Reply:
True, this can be removed.

Reply:
done